definitions/npm/ramda_v0.26.x/flow_v0.76.x-v0.103.x/test_ramda_v0.26.x_list.js

```diff
@@ -703,4 +703,40 @@ const str: string = "hello world";
     s: a,
     y: b
   }))([1, 2, 3])(["1", "2", "3"]);
+
+  describe('uniq', () => {
```

Comment:
These cases will need similar love since they are a copy/pasta of the other tests.

Reply:
Done